packages/remirror__theme/src/utils.ts

```diff
@@ -365,6 +365,7 @@ export const defaultRemirrorTheme: Remirror.Theme = {
     shadow3,
     backdrop: transparentize(foreground, 0.1),
     outline: transparentize(primary, 0.6),
+    divider: lighten(foreground, 0.8),
```

Comment:
highlight for reviewers: I added a new `divider` value into the `Remirror.Theme`